src/components/NavBar/index.tsx

```diff
@@ -23,35 +23,37 @@ const NavBar = () => {
 
   return (
     <Navbar expand="lg" collapseOnSelect className="navbar">
-      <LinkContainer to='/'>
+      <LinkContainer to="/">
         <Navbar.Brand>TinDev</Navbar.Brand>
       </LinkContainer>
       <Navbar.Toggle aria-controls="basic-navbar-nav" />
       <Navbar.Collapse id="basic-navbar-nav">
         <Nav className="ml-auto">
-          {userInfo && 
-          <LinkContainer to='/register'>
-            <Nav.Link><i className="register"></i>Register</Nav.Link>
-          </LinkContainer>}
-          {userInfo ? (
-          <NavDropdown title={userInfo.userName} id="username">
-            <LinkContainer to='/profile'>
-              <NavDropdown.Item>Profile</NavDropdown.Item>
+          {!userInfo && (
+            <LinkContainer to="/register">
+              <Nav.Link>
+                <i className="register"></i>Register
+              </Nav.Link>
             </LinkContainer>
-            <LinkContainer to={`/userpage/${userInfo.id}`}>
-              <NavDropdown.Item>Home</NavDropdown.Item>
```

Comment:
No link to user homepage? I didn't see a link in any pages in figma, so I added it to the navbar.